MemoryManagement/MemoryManagement/AppDelegate.h

```diff
@@ -0,0 +1,15 @@
+//
+//  AppDelegate.h
+//  MemoryManagement
+//
+//  Created by Lambda_School_Loaner_268 on 5/20/20.
+//  Copyright © 2020 Lambda. All rights reserved.
+//
+
+#import <UIKit/UIKit.h>
+
+extern NSMutableArray *artists;
```

Comment:
This (having a variable globally accessible across the app) is generally not a suggested practice. Having a "shared state" accessible across the app can make things hard to debug and test and can lead to unexpected issues if you're not super careful. Having a model controller that we pass around will help avoid these issues and keep things tidier and help us track down bugs more easily.